app/src/main/java/org/hackillinois/android/view/home/eventlist/EventsSection.kt

```diff
@@ -73,6 +73,7 @@ class EventsSection(
     }
 
     fun updateEventsList(newEventsList: List<Event>) {
+        FavoritesManager.updateFavoriteNotifications(context, eventsList, newEventsList)
```

Comment:
It might be better to do this in the EventRepository, since this code will only run on the homepage event list. If it is in the EventRepository, it will be run when the events list is updated from the server (which may happen on another page)

Reply:
All the functions in `FavoritesManager` take in a context as a parameter. How can we pass in that context from the `EventsRepository`, since nothing in `refreshAllEvents` takes context as parameter?

Reply:
Hmm, good point. I believe there is a way to get a context in the `ViewModel` by changing it to an `AndroidViewModel`

This might be bad design though, so maybe this is better.